src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingHome.java

```diff
@@ -198,6 +253,118 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
         }
     }
 
+    /**
+     * Try to spawn a new citizen as child.
+     */
+    public void trySpawnChild()
+    {
+        // Spawn a child when adults are present and the house has space, so level 3+
+        if (colony.canMoveIn() && femalePresent && malePresent && colony.getCitizenManager().getCurrentCitizenCount() < colony.getCitizenManager().getMaxCitizens())
+        {
+            CitizenData mom = null;
+            CitizenData dad = null;
+
+            for (final CitizenData data : getAssignedCitizen())
+            {
+                if (data.isFemale() && !data.isChild())
+                {
+                    mom = data;
+                }
+                else if (!data.isChild())
+                {
+                    dad = data;
+                }
+
+                if (mom != null && dad != null)
+                {
+                    break;
+                }
+            }
+
+            if (mom == null || dad == null)
+            {
+                return;
+            }
+
+            final CitizenData newCitizen = colony.getCitizenManager().createAndRegisterNewCitizenData();
+
+            final Random rand = new Random();
+
+            // Inheriting stats from parents + some randomness. Capped by happiness
+            int str = (mom.getStrength() + dad.getStrength()) / 2 + rand.nextInt(3) - rand.nextInt(3);
```

Comment:
Personally i feel that a child should have a bigger change of becoming better then worse then his parents.
I propose doing rang.nextInt(5) - rand.nextInt(3). Also i would like to see these in constants, so that they can be controlled better.

Reply:
Sounds good yes

Reply:
hm that might become too much with the school thats going to be added for boosting child stats

Reply:
and longterm it would inflate the citizens stats, as each generation would have more so you get to max stats pretty fast. Which is why I made it avg +- random so you can still benefit from manually selecting parents but it has some variance

Reply:
Agreed

Reply:
Then make the chance go down slowly. The idea here is that citizen learn from their parents.

Reply:
well they do learn from their parents through the avg stats and the +/-3 is kinda like their innate natural ability which can vary